src/main/java/uk/gov/register/core/Item.java

```diff
@@ -17,25 +17,42 @@
     private static final CanonicalJsonMapper canonicalJsonMapper = new CanonicalJsonMapper();
 
     private final JsonNode content;
-    private final HashValue hashValue;
+    private final HashValue oldHashValue;
+    private final HashValue blobHash;
 
     public Item(JsonNode content) {
-        this(itemHash(content), content);
+        this(itemHash(content), objectHash(content), content);
     }
 
-    public Item(HashValue hashValue, JsonNode content) {
-        this.hashValue = hashValue;
+    public Item(HashValue oldHashValue, HashValue blobHash, JsonNode content) {
+        this.oldHashValue = oldHashValue;
+        this.blobHash = blobHash;
         this.content = content;
     }
 
+    @Deprecated
+    public Item(HashValue oldHashValue, JsonNode content) {
+        // This should be removed
```

Comment:
I can either remove this now (in a follow up PR), or leave it until we remove V1.

It's dangerous to leave it in, because it means we are creating items with the new hash set wrongly (even if its only in tests)

The reason I haven't changed it here in this PR is because it requires a lot of minor test changes, and it will clutter up the review

Reply:
I think this is OK for now but can you add a `TODO` to the top the comment so it is flagged by the IDE.

Reply:
No problem, have added this